src/app/reports/utility/immunised.html

```diff
@@ -7,12 +7,12 @@ <h5 class="pull-left">Average Immunized </h5>
   <h3 class="pull-right">| {{utilityCtrl.kpiAgg.ave}}</h3>
 </div>
 <div class="btn btn-success btn-lg clearfix">
-  <h5 class="pull-left">Highest Immunized <br/> ({{utilityCtrl.rounds[utilityCtrl.kpiValues[0][0]]}})</h5>
-  <h3 class="pull-right">| {{utilityCtrl.kpiValues[0][1]}}</h3>
+  <h5 class="pull-left">Highest Immunized <br/> ({{utilityCtrl.rounds[utilityCtrl.kpiValues[utilityCtrl.kpiValues.length -1][0]]}})</h5>
```

Comment:
Yeah. So the function sorts an array of Int from highest to lowest, so ````utilityCtrl.rounds[utilityCtrl.kpiValues[0][0]]```` is the first child of the array, representing the highest value and  ````utilityCtrl.rounds[utilityCtrl.kpiValues[utilityCtrl.kpiValues.length -1][0]]```` the lowest value.